modules/distribution/src/sso/views/register_new_user.jag

```diff
@@ -0,0 +1,200 @@
+                                        </div>
+                                    <%
+                                            renderedClaims[formElementName] = claims[claimIndex].claimUri;
+                                        }
+                                    }
+                                    %>
+
+                                    <input type="hidden" name="reg-claims" value='<%=renderedClaims%>'
+                                       class="input-block-level required">
+                                    <input id="reg-action" name="action" type="hidden" value="register">
+                                    <input id="reg-relyingParty" name="relyingParty" type="hidden" value="<%=relyingParty%>">
+                                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 form-group">
+                                    <br/><br/>
+                                    <button id="registrationSubmit" class="wr-btn grey-bg col-xs-12 col-md-12 col-lg-12 uppercase font-extra-large"
+                                            type="button">Register
+                                    </button>
+                                </div>
+                                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 form-group">
+                                    <span class="margin-top padding-top-double font-large">Already have an account? </span>
+                                    <a href="<%=signInUrl%>" id="signInLink" class="font-large">Sign in</a>
+                                </div>
+                                <div class="clearfix"></div>
+                                </div>
+                            </div>
+                        </form>
+                    </div>
+    </div>
+</div>
+
+ <% include(themeModule.getThemedUrl('includes/footer_file_imports.jag')); %>
+<script type="text/javascript">
+    $(function () {
+        var ENTER_KEY = 13;
+        function register() {
+            if (!$('#registerForm').valid()) {
+                return;
+            }
+
+            var formData = $("#registerForm").serialize();
+
+            $.post('/sso/register', formData, function (data) {
+                var regFormError = $('#regFormError');
+                var regFromSuccess = $('#regFormSuc');
+                if (JSON.parse(data).error == 'false') {
+                    regFormError.hide();
+                    regFromSuccess.show();
+                    regFromSuccess.text(JSON.parse(data).msg + ". You will be redirected to login page.");
+                    setTimeout(function () {
+                        window.location = "<%=signInUrl%>";
+                    }, 2000);
+                } else {
+                    regFromSuccess.hide();
+                    regFormError.show();
+                    regFormError.text(JSON.parse(data).msg);
+                }
+            });
+        }
+
+        $("#registrationSubmit").bind('click', register);
```

Comment:
Be consistent on using quotes for string definitions. use double quotes as the standard.